packages/core/src/customer/strategies/stripe-upe/stripe-upe-customer-strategy.spec.ts

```diff
@@ -203,7 +203,7 @@ describe('StripeUpeCustomerStrategy', () => {
         });
 
         it('returns an error when methodId is not present', () => {
-            const promise = strategy.initialize({ ...getStripeUPECustomerInitializeOptionsMock(), methodId: '' });
+            const promise = strategy.initialize({ ...getStripeUPECustomerInitializeOptionsMock(), methodId: undefined });
```

Comment:
```suggestion
            const promise = strategy.initialize({ ...getStripeUPECustomerInitializeOptionsMock()});
```

Reply:
Had to do this for the test, I have made relevant changes to the strategy as well.